mlflow/sagemaker/__init__.py

```diff
@@ -1336,6 +1336,22 @@ def _get_sagemaker_config_name(endpoint_name):
     return f"{endpoint_name}-config-{get_unique_resource_id()}"
 
 
+def _get_sagemaker_config_tags(endpoint_name):
+    return [{"Key": "app_name", "Value": endpoint_name}]
+
+def _prepare_sagemaker_tags(
+  sagemaker_tags: Dict[str, str],
```

Comment:
What happens if the dict ``sagemaker_tags`` contains the key ``"app_name"``? What does AWS do with multiple conflicting tags?